src/imageApiCalls.js

```diff
@@ -0,0 +1,9 @@
+export const getImages = () => {
+  return fetch('https://picsum.photos/v2/list')
+    .then(response => {
+      if(response.ok) {
+        return response.json()
+      }
```

Comment:
Would like to see some error handling around this for sure! Maybe if the response is not okay, have a back up photo?

Reply:
I added an error handle in the imagesData.tsx so that an error logo shows up when the the useEffect catches an error.